ciao-controller/types/types.go

```diff
@@ -87,6 +87,7 @@ type Instance struct {
 	CNCI        bool                `json:"-"`
 	Usage       map[string]int      `json:"-"`
 	Attachments []StorageAttachment `json:"-"`
+	CreateTime  time.Time           `json:"create_time"`
```

Comment:
@markdryan and I couldn't find where that struct was marshalled out at all. I'll update the patch to not marshal this one and open an issue about removing the marshalling if not necessary.